manifests/broker.pp

```diff
@@ -69,6 +72,7 @@
   $group_id                                  = $kafka::params::group_id,
   $user_id                                   = $kafka::params::user_id,
   $config_dir                                = $kafka::params::config_dir,
+  $bin_dir                                   = $kafka::params::bin_dir,
```

Comment:
@LionelCons can you use the datatype Stdlib::Absolutepath here and in all other locations where you use the new parameter?